cumulusci/tasks/bulkdata/generate_from_yaml.py

```diff
@@ -159,6 +159,7 @@ def generate_data(self, dburl, num_records, current_batch_num):
                 dburl=dburl,
                 load_declarations=self.loading_rules,
                 should_create_cci_record_type_tables=True,
+                plugin_options={"orgname": self.org_config.name},
```

Comment:
What is this used for?

Reply:
Makes it possible for Snowfakery to call back into Salesforce for SOQL queries.

Reply:
The way that Snowfakery is setting up a CumulusCI runtime using CliRuntime(load_keychain=True) to do this is not going to work in the context of MetaCI which doesn't use a filesystem-based keychain. It would probably work out better to make Snowfakery take an org config instance instead of just a name.

Reply:
Hmmm...yes, I'd forgotten about the MetaCI Keychain. Okay I'll need to update Snowfakery.

Reply:
get_simple_salesforce_connection takes a project_config and an org_config. So do I have to pass BOTH of those from CumulusCI to Snowfakery if I want to be able to do authtoken refreshes later?

Reply:
https://github.com/SFDO-Tooling/Snowfakery/pull/430